### DISCUSSION TITLE
API commands list for coredns

### DISCUSSION THREAD

**Discussion by sajiljosephs** [coredns/coredns#6447] [General]
I read everywhere that coredns supports api commands to integrate it from an application. Do we have a list somewhere on the list of API commands ? I am not talking about Kubernetes related API commands

**Comment by chrisohaver** [comment]
CoreDNS doesn’t serve an API.

**Reply by chrisohaver** [reply]
You may be reading about plugin development, for which you can develop your own plugin by writing code that implements the coredns plugin interface, and building a custom coredns binary with your new plugin.

**Comment by chrisohaver** [comment]
https://coredns.io/2016/12/19/writing-plugins-for-coredns/
